packages/admin-ui-core/src/values/styleKit.ts

```diff
@@ -34,15 +44,21 @@ export function listBoxItem(tone: 'main' | 'critical', selected = false) {
   }
 }
 
-export function focusVisible(tone: Tone): StyleProp {
+export function focusVisible(
+  toneOrColorToken: Tone | ColorTokens,
+  focusProps?: StyleProp,
+  polyfillProps?: StyleProp
```

Comment:
What do you think of something like that? I think it's better than creating two parameters
```suggestion
  styleProps?: { polyfill?: StyleProp, focus?: StyleProp }
```

Reply:
Looks good to me!